Tools/parametric_model/configs/fixedwing_longitudinal_model.yaml

```diff
@@ -0,0 +1,164 @@
+# general information
+model_name: "Gazebo Standard Plane"
+model_type: "Standard Plane Longitudinal Model"
+model_class:
+  "SimpleFixedWingModel"
+
+  # all vectors in FRD body frame if not specified otherwise
+model_config:
+  mass: 20.0
```

Comment:
Why is the vehicle so heavy? Is this because this is coming from some paper?

Reply:
Honestly, I think I just inserted some random values as they only scale the resulting parameters anyways and should not influence tracking. I'm going to look into values that make more sense.